### DISCUSSION TITLE
So cool! Effects stop when the terminal is closed?

### DISCUSSION THREAD

**Discussion by gutierrezphd** [4JX/L5P-Keyboard-RGB#108] [Q&A]
This is way cool! 

Is there a way to keep the dynamic effects running after closing the GUI or the terminal window in which the program is running?

**Comment by 4JX** [comment]
v0.18.0 should work for that, but can you test https://github.com/4JX/L5P-Keyboard-RGB/actions/runs/4746596729? Closing the window itself when using the GUI should leave the effect playing.

**Reply by HyperN00B** [reply]
Yeah closing the GUI keeps it running. But killing the terminal used for starting the app itself causes the animation to stop. (Using nohup and & fixes it by keeping the program alive even after the parent program dies) Maybe a .service can keep it seemlessly running
